users/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+
+class App extends Component {
+	constructor() {
+		super();
+		this.state = {
+			users: [],
+			name: '',
+			bio: '',
+			id: '',
+			isEditing: false,
+		};
+	}
+
+	componentDidMount() {
+		this.getUsers();
+	}
+	handleChanges = e => {
+		e.preventDefault();
+		this.setState({
+			[e.target.name]: e.target.value,
+		});
+	};
+
+	getUsers = () => {
+		axios.get('http://localhost:5000/api/users').then(res => {
+			this.setState({ users: res.data.users });
+		});
+	};
+	deleteUser = (e, id) => {
+		e.preventDefault();
+		axios.delete(`http://localhost:5000/api/users/${id}`).then(res => {
+			console.log(res);
+			axios.get('http://localhost:5000/api/users').then(res => {
+				this.setState({ users: res.data.users });
+			});
+		});
+	};
+
+	addUser = e => {
+		e.preventDefault();
+		let user = { name: this.state.name, bio: this.state.bio };
+		axios.post(`http://localhost:5000/api/users/`, user).then(res => {
+			console.log(res);
+			axios.get('http://localhost:5000/api/users').then(res => {
+				this.setState({ users: res.data.users });
+			});
+		});
+	};
+
+	updateUser = (e, user) => {
+		e.preventDefault();
+		console.log('update user');
+		this.setState({
+			name: user.name,
+			bio: user.bio,
+			id: user.id,
+			isEditing: true,
+		});
+	};
+	submitUser = e => {
+		console.log('submitUser');
+		e.preventDefault();
+		// let id = this.state.id;
+		let changed = { name: this.state.name, bio: this.state.bio };
+		axios.put(`http://localhost:5000/api/users/${this.state.id}`, changed).then(res => {
+			console.log('put response', res);
+			axios.get('http://localhost:5000/api/users').then(res => {
+				this.setState({ users: res.data.users });
+			});
+		});
+		this.setState({ isEditing: false });
```

Comment:
To get this part of your app to work, look into promises and using them when chaining axios calls together like this